static/src/javascripts/projects/common/modules/experiments/tests/adblocking-response.js

```diff
@@ -73,13 +73,13 @@ define([
             id: 'control',
             test: function(){}
         }, {
-            id: '299',
+            id: '300',
             test: function () {
                 if (isQualified()) {
```

Comment:
After chatting in person, what you've got makes sense, there's no need to change this.